defaults/main.yml

```diff
@@ -12,7 +12,8 @@ sensu_enterprise_package: sensu-enterprise
 sensu_enterprise_dashboard_package: sensu-enterprise-dashboard
 
 # Sensu repo urls
-sensu_yum_repo_url: "http://repositories.sensuapp.org/yum/$releasever/$basearch/"
+sensu_yum_repo_url: "https://repositories.sensuapp.org/yum/$releasever/$basearch/"
```

Comment:
Is this supposed to be changed to `https://sensu.global.ssl.fastly.net/` instead?